src/frontend/src/lib/rest/blockchain.rest.ts

```diff
@@ -0,0 +1,27 @@
+import type { BitcoinAddressData, BlockchainBtcAddressDataParams } from '$lib/types/blockchain';
+
+const API_URL = import.meta.env.VITE_BLOCKCHAIN_API_URL;
+
+/**
+ * Get BTC address data (including transactions).
+ *
+ * Documentation:
+ * - https://www.blockchain.com/explorer/api/blockchain_api
+ *
+ */
+export const btcAddressData = async ({
```

Comment:
Likewise, didnt' we reviewed this already? Sorry if not

Reply:
The branch needed rebasing, unrelated changes are gone now.

Reply:
Thanks, I was confused.